test/extended/util/framework.go

```diff
@@ -938,6 +940,34 @@ func WaitForServiceAccount(c corev1client.ServiceAccountInterface, name string)
 	return wait.Poll(time.Duration(100*time.Millisecond), 3*time.Minute, waitFn)
 }
 
+// WaitForNamespaceSCCAnnotations waits until the cluster-policy-controller pods in the openshift-kube-controller-manager
+// namespace have added the SCC related annotations to the provided namespace;  NOTE: per
+// https://github.com/openshift/cluster-policy-controller/blob/master/pkg/security/controller/namespace_scc_allocation_controller.go#L74
+// the relist interval of that controller is 10 minutes, so are wait time here works off of that
```

Comment:
comment nit - can probably clean this up per discussion in this thread:
```suggestion
// WaitForNamespaceSCCAnnotations waits up to 2 minutes for the cluster-policy-controller to add the SCC related
// annotations to the provided namespace.
```